custom_components/aguaiot/const.py

```diff
@@ -130,6 +152,16 @@ class AguaIOTSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=NumberDeviceClass.TEMPERATURE,
     ),
+    NumberEntityDescription(
```

Comment:
This is already implemented as fan mode in the climate entity. Why did you choose to implement it as a separate control too?

Reply:
Due to the fat that I have to implement power control for wood and pellet. Implementing a second climate control when the stove switch to wood will be a non-sense.